nilearn/input_data/nifti_masker.py

```diff
@@ -186,7 +186,7 @@ def fit(self, imgs=None, y=None):
             self.affine_ = self.mask_img_.get_affine()
         # Load data in memory
         self.mask_img_.get_data()
-        if self.verbose > 10:
+        if self.verbose > 0:
```

Comment:
Indeed, I think that it should be a high value. Printing just before a return is not someone wants often: most often we know that the operation is finished because either another one is starting, or the processing has finished.